Traceback (most recent call last):
  File "c:\Users\iacop\Desktop\Programmazione\Github\tum-adlr-11\2D_Shape_Completion\main.py", line 81, in <module>
    criterion = nn.CrossEntropyLoss()
                ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\iacop\anaconda3\Lib\site-packages\torch\__init__.py", line 2562, in __getattr__
    raise AttributeError(f"module '{__name__}' has no attribute '{name}'")
AttributeError: module 'torch' has no attribute 'CrossEntropyLoss'
